pkgs/by-name/go/gowall/package.nix

```
@@ -0,0 +1,31 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "gowall";
+  version = "0.1.5";
+
+  src = fetchFromGitHub {
+    owner = "Achno";
+    repo = "gowall";
+    rev = "v${version}";
+    hash = "sha256-4h7vRu1aqCGccKx2UiLSFNloqf22QUml4BHkKzzdwYA=";
+  };
+
+  vendorHash = "sha256-jNx4ehew+IBx7M6ey/rT0vb53+9OBVYSEDJv8JWfZIw=";
+
+  meta = {
+    changelog = "https://github.com/Achno/gowall/releases/tag/v${version}";
```

Would using `src.rev` be beneficial here?
```suggestion
    changelog = "https://github.com/Achno/gowall/releases/tag/${src.rev}";
```